packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -546,7 +545,7 @@ const MultiSelect = (props: Props) => {
         const menuText = getMenuText(allChildren);
 
         const dropdownOpener = (
-            <IDProvider id={id} scope="multi-select-opener">
+            <Id id={id}>
```

Comment:
**nit:** This is now a function component, so you could use `useId` directly as well. I understand if you want to keep things simple for this migration.

Reply:
Yeah, I considered that, but the component seems so complex; this felt cleaner and a little easier. :shrug:

Reply:
makes total sense :)